<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WombatForensics</class>
 <widget class="QMainWindow" name="WombatForensics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Wombat Forensics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabSpacing">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>226</width>
           <height>392</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="label">
          <string>Directory Tree</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeView" name="directoryTreeView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="headerDefaultSectionSize">
               <number>0</number>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>226</width>
           <height>392</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="label">
          <string>File Type</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTreeView" name="fileSignatureTreeView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>226</width>
           <height>392</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="label">
          <string>Search Results</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>226</width>
           <height>392</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="label">
          <string>Bookmarks</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QTabWidget" name="fileViewTabWidget">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="naturalTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Natural</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="hexTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Hex</string>
           </attribute>
           <widget class="QLabel" name="testLabel">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>60</y>
              <width>331</width>
              <height>91</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QColumnView" name="columnView">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;DejaVu Sans Mono&quot;;</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEvidence">
    <property name="title">
     <string>Evidence</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEvidence"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Case"/>
   <addaction name="actionOpen_Case"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Evidence"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Case">
   <property name="icon">
    <iconset resource="wombat.qrc">
     <normaloff>:/newcase</normaloff>:/newcase</iconset>
   </property>
   <property name="text">
    <string>New Case</string>
   </property>
   <property name="toolTip">
    <string>Create a New Case</string>
   </property>
  </action>
  <action name="actionOpen_Case">
   <property name="icon">
    <iconset resource="wombat.qrc">
     <normaloff>:/opencase</normaloff>:/opencase</iconset>
   </property>
   <property name="text">
    <string>Open Case</string>
   </property>
   <property name="toolTip">
    <string>Open an Existing Case</string>
   </property>
  </action>
  <action name="actionAdd_Evidence">
   <property name="icon">
    <iconset resource="wombat.qrc">
     <normaloff>:/addevidence</normaloff>:/addevidence</iconset>
   </property>
   <property name="text">
    <string>Add Evidence</string>
   </property>
   <property name="toolTip">
    <string>Add Evidence to the Case</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="wombat.qrc"/>
 </resources>
 <connections/>
</ui>
